### ISSUE TITLE
Brave crashes  nvidia-drm on Linux when running with the proprietary driver 

### ISSUE THREAD

**Issue by bhlevca** [brave/brave-browser#14280]
## Description 

After and only after starting the brave browser the following errors appear in the error logs:
```
Feb 21 10:36:42 high kernel: [drm:nv_drm_gem_fence_attach_ioctl [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to lookup gem object for fence attach: 0x0000000a
Feb 21 10:36:42 high kernel: [drm:nv_drm_gem_fence_attach_ioctl [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to lookup gem object for fence attach: 0x0000001f
Feb 21 10:36:42 high kernel: [drm:nv_drm_gem_fence_attach_ioctl [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to lookup gem object for fence attach: 0x00000016
Feb 21 10:36:42 high kernel: [drm:nv_drm_gem_fence_attach_ioctl [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to lookup gem object for fence attach: 0x00000012
Feb 21 10:36:42 high kernel: [drm:nv_drm_gem_fence_attach_ioctl [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to lookup gem object for fence attach: 0x0000002a
Feb 21 10:36:42 high kernel: [drm:nv_drm_gem_fence_attach_ioctl [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to lookup gem object for fence attach: 0x00000004
Feb 21 10:36:42 high kernel: [drm:nv_drm_gem_fence_attach_ioctl [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to lookup gem object for fence attach: 0x00000020
Feb 21 10:36:42 high kernel: [drm:nv_drm_gem_fence_attach_ioctl [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to lookup gem object for fence attach: 0x00000022
Feb 21 10:36:42 high kernel: [drm:nv_drm_gem_fence_attach_ioctl [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to lookup gem object for fence attach: 0x00000021
Feb 21 10:36:42 high kernel: [drm:nv_drm_gem_fence_attach_ioctl [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to lookup gem object for fence attach: 0x00000004
```
The X11 session can be still used to some extent but certain functions are disabled such as starting certain programs from the icons on the taskbar and shutting down the X11 session from the desktop. Only `shutdown -P` works

## Steps to Reproduce

   1.  Start the X11 session 
   2.  Monitor the logs 
   3.  start various programs and look at the logs as no errors occur
   4. Start  brave browser
   5. Notice in the logs the appearance of the `[drm:nv_drm_gem_fence_attach_ioctl [nvidia_drm]] *ERROR*`
   6. Notice the partial disabling of the desktop session

## Actual result:

`[drm:nv_drm_gem_fence_attach_ioctl [nvidia_drm]] *ERROR*`

## Expected result:
Expect no issues. For example, starting firefox does not cause the issue

## Reproduces how often: 

Every single time

## Version/Channel Information:

```

Brave | 1.20.108 Chromium: 88.0.4324.182 (Official Build) (64-bit)
-- | --
Revision | 73ee5087001dcef33047c4ed650471b225dd8caf-refs/branch-heads/4324@{#2202}
OS | Linux
JavaScript | V8 8.8.278.17
User agent | Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.182 Safari/537.36
Command Line | /opt/brave.com/brave/brave --enable-dom-distiller --disable-domain-reliability --no-pings --extension-content-verification=enforce_strict --extensions-install-verification=enforce --origin-trial-public-key=bYUKPJoPnCxeNvu72j4EmPuK7tr1PAC7SHh8ld9Mw3E=,fMS4mpO6buLQ/QMd+zJmxzty/VQ6B1EUZqoCU04zoRU= --sync-url=https://sync-v2.brave.com/v2 --lso-url=https://no-thanks.invalid --variations-server-url=https://variations.brave.com/seed --enable-features=WebUIDarkMode,LegacyTLSEnforced,DnsOverHttps,PrefetchPrivacyChanges,PasswordImport,ReducedReferrerGranularity,AutoupgradeMixedContent,SafetyTip --disable-features=AutofillEnableAccountWalletStorage,AutofillServerCommunication,DirectSockets,TextFragmentAnchor,NotificationTriggers,WebOTP,PrivacySettingsRedesign,SignedExchangePrefetchCacheForNavigations,SubresourceWebBundles,LangClientHintHeader,PasswordCheck,NetworkTimeServiceQuerying,TabHoverCards,IdleDetection,SignedExchangeSubresourcePrefetch,SafeBrowsingEnhancedProtection --flag-switches-begin --load-media-router-component-extension=1 --flag-switches-end
Executable Path | /opt/brave.com/brave/brave
Profile Path | /home/bogdan/.config/BraveSoftware/Brave-Browser/Default
```

- Can you reproduce this issue with the current release?   YES
- Can you reproduce this issue with the beta channel?   did not try
- Can you reproduce this issue with the nightly channel?  did not try 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?  NO
- Does the issue resolve itself when disabling Brave Rewards?  NO
- Is the issue reproducible on the latest version of Chrome?  YES 
- 
## Miscellaneous Information:

The issue may be on Chrome/Chromium end.

**Comment by bsclifton** [comment]
@bhlevca there definitely are problems with drivers, especially on Linux. You'll want to disable hardware acceleration for now - that should get you sorted out (at the expense of things rendering in software):
brave://settings/system > `Use hardware acceleration when available`

**Comment by bhlevca** [comment]
I tried disabling the hardware acceleration, the problem is still there. I will wait until I will get the Radeon card and hope that their drivers are better.

**Comment by bsclifton** [comment]
Hi folks - I am going to close this issue as it's a bit old without an update

I did put together some notes and a [new process for when folks are having GPU issues](https://github.com/brave/brave-browser/wiki/GPU-issues---incident-reports).

If you're reading this and you are having GPU issues, you can help me by creating a new issue here describing the problem on GitHub and then attaching the report from brave://gpu.
